test/regress/CMakeLists.txt

```diff
@@ -2859,9 +2859,10 @@ add_custom_target(regress
 
 macro(cvc5_add_regression_test level file)
   add_test(${file}
+    ${PYTHON_EXECUTABLE}
     ${run_regress_script}
     ${RUN_REGRESSION_ARGS}
-    ${path_to_cvc5}/cvc5 ${CMAKE_CURRENT_LIST_DIR}/${file})
+    ${path_to_cvc5}/cvc5${CMAKE_EXECUTABLE_SUFFIX} ${CMAKE_CURRENT_LIST_DIR}/${file})
```

Comment:
Can we instead get the output filename of the `cvc5-bin` target? I guess we can get it via `get_target_property(... cvc5-bin OUTPUT_NAME)`

Reply:
It seems that, experimentally, `OUTPUT_NAME` gives `cvc5` and not `cvc5.exe` 🤷 

I tided-up the logic a bit to make the intent more explicit.